blog/migrations/__init__.py

```diff
@@ -0,0 +1 @@
+"""This package is for managing database schema migrations."""
```

Comment:
No, `manage.py` commands are for managing migrations. This package only contains the migration scripts but doesn't magically invoke them by itself, when not asked to.

Reply:
Edited.

Reply:
Marking as resolved.